### ISSUE TITLE
After CCleaner use firefox settings are lost!

### ISSUE THREAD

**Issue by rezpower** [zombieFox/nightTab#209]
There have been posts about firefox loosing it's settings before. In my case every time I use CCleaner to clean my browser and PC nighttab will loose all it's settings. This dose not happen with Edge Chromium while ccleaner also cleans edge data.
Of course no one can blame nighttab for this as the problem is comming from ccleaner. Just want to know if anyone knows what should one add to ccleaner exclude list to prevent this from happening. ( a cookie or a file?!)

**Comment by rezpower** [comment]
Ok I may sound stupid answering to myself! But I have been looking into this and here is what I found. nighttab uses a cookie and if this cookie is deleted on you next opening of firefox you will loose your settings. In my case the name of the cookie is:
**0d88aaf6-ee1a-4cb6-af64-9bb99bdf514d**
I added this cookie to CCleaner exceptions and now the problem is fixed! 
I also use another extension called "cookie auto delete". I also added the nighttab cookie to it's white list and all works perfect now :)

**Comment by zombieFox** [comment]
Yes, nightTab stores all the user added data in [Local Storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage). So apps or [other extensions](https://github.com/zombieFox/nightTab/issues/205) wipe browser cache or cookies they often include Local Storage. Glad you found a way to exclude the data from CCleaner. 👍

**Comment by rezpower** [comment]
Thanks for your response zombiefox.

For those using CCleaner here is how to exclude nighttab data:
![Annotation 2020-07-31 144746](https://user-images.githubusercontent.com/28585954/89039461-67bd8f80-d342-11ea-98d6-8c21add81c04.png)

remember to do the same thing in any other addon that you use to clean your browser.

**Comment by ktnlikeasmallcat** [comment]
I'm having a similar issue with firefox preferences' exceptions - cookies and site data. I've blocked nytimes and bing cookies, which appear to be coming from nightTab (I could be totally wrong). and I've allowed moz-extension://f7653fc7-ff3f-41af-bbdc-08c31864815a/index.html as well as http(s)://f7653fc7-ff3f-41af-bbdc-08c31864815a. Everytime I reboot, nighttab resets to default settings. I've been just opening a json backup but if there's a better fix I don't know it. This is the best new tab I've ever had!
